CONTRIBUTING.md

```diff
@@ -32,85 +32,29 @@ Here are a few things you can do that will increase the likelihood of your pull
 Make sure you have a fairly recent version of vscode (>1.32) and are using nodejs
 version >=v10.13.0. (Tested on v10.15.1 and v10.16.0).
 
-This repo uses [Rush](https://rushjs.io) to handle package management, building, and other
-operations across multiple projects. See the Rush "[Getting started as a developer](https://rushjs.io/pages/developer/new_developer/)" docs
-for more details.
-
-If you plan on building from the command line, it's easiest if Rush is installed globally:
-
-```shell
-npm install -g @microsoft/rush
-```
-
-To get started, run:
-
-```shell
-rush update && rush build
-```
-
-Note that when you run the `rush` command from the globally installed version, it will examine the
-`rushVersion` property in the repo's `rush.json`, and if it differs from the globally installed
-version, it will download, cache, and run the version of Rush specified in the `rushVersion`
-property.
-
-A few more things to know about using rush:
-
-* Avoid running `npm` for any commands that install/link dependencies
-* Instead use the *rush* equivalent: `rush add <package>`, `rush update`, etc.
-* If you plan on only building via VS Code tasks, you don't need Rush installed at all, since those
-tasks run `common/scripts/install-run-rush.js` to bootstrap a locally installed and cached copy of
-Rush.
-
 ### Building
```

Comment:
Nice to see the build process simplified 😃 

I find the current headings a bit confusing though. I can't add suggestions for some reason, but how about replacing lines 35-59 ("### Building" -> "### Installing") with something like this? 👇🏼 (If I've interpreted the instructions properly!) 

--- 
### Installing all packages

From the command line, go to the directory `extensions/ql-vscode` and run

```shell
npm install
```

### Building the extension

From the command line, go to the directory `extensions/ql-vscode` and run

```shell
npm run build
```

Alternatively, you can build the extension within VS Code via `Terminal > Run Build Task...` (or `Ctrl+Shift+B` with the default key bindings).

### Installing the extension

...

Reply:
I like that better, thanks!